api/android/README.md

```diff
@@ -18,23 +18,23 @@ We assume that you already have experienced Android application developments wit
 First of all, you need to set-up the development environment as following:
 
 ```bash
-$ export ANDROID_DEV_ROOT=$HOME/android               # Set your own path (The default path will be "$HOME/Android".)
+$ export ANDROID_DEV_ROOT=$HOME/myandroid           # Set your own path (default location: $HOME/Android)
 $ mkdir -p $ANDROID_DEV_ROOT/tools/sdk
 $ mkdir -p $ANDROID_DEV_ROOT/tools/ndk
 $ mkdir -p $ANDROID_DEV_ROOT/gstreamer-1.0
 $ mkdir -p $ANDROID_DEV_ROOT/workspace
 $
 $ vi ~/.bashrc
-# Environment variables for developing a NNStreamer application
+# The environment variables to develop an Android application with NNStreamer
 #
 export JAVA_HOME=/opt/android-studio/jre            # JRE path in Android Studio
-export ANDROID_DEV_ROOT=$HOME/android               # Set your own path (The default path will be "$HOME/Android".)
-#
-# $ANDROID_DEV_ROOT/tools/sdk                    # Android SDK root directory (default location: $HOME/Android/Sdk)
-# $ANDROID_DEV_ROOT/tools/ndk                    # Android NDK root directory (default location: $HOME/Android/Sdk/ndk/<ndk-version>)
-# $ANDROID_DEV_ROOT/gstreamer-1.0                # GStreamer binaries
-# $ANDROID_DEV_ROOT/workspace/nnstreamer         # NNStreamer cloned git repository
-#
+export ANDROID_DEV_ROOT=$HOME/android               # Set your own path (default location: "$HOME/Android".)
+
+# $ANDROID_DEV_ROOT/tools/sdk/: Android SDK root directory (default location: $HOME/Android/Sdk)
```

Comment:
Q about (default location: $HOME/Android/Sdk):

Those default locations are the directories which the Android SDK and NDK are installed in by default? I am confusing.